api/src/main/java/org/openmrs/module/fhir2/providers/r3/MedicationRequestFhirResourceProvider.java

```diff
@@ -49,4 +60,28 @@ public MedicationRequest getMedicationRequestById(@IdParam @NotNull IdType id) {
 		
 		return MedicationRequest30_40.convertMedicationRequest(medicationRequest);
 	}
+
+	@Search
```

Comment:
This should initially be developed in the R4 ResourceProvider and then added to R3 afterwards.

Reply:
Do you mean I shouldnt write code in the R3 providers? Because I have tried implementing it in both

Reply:
No, I just don't see the R4 provider as part of this PR?